src/index.html

```diff
@@ -20,8 +20,312 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+          width="40"
+          height="40"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
```

Comment:
Add classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```